pilotage/model/stage.go

```diff
@@ -0,0 +1,22 @@
+package model
+
+import "time"
+
+type StageV1 struct {
+	ID         string
+	FlowID     string
+	StageType  string
+	Name       string
+	Title      string
+	Sequencing string
+	CreateAt   time.Time
+	UpdateAt   time.Time
+}
+
+type StageDataV1 struct {
+	ID      string
+	StageID string
```

Comment:
@genedna  I don't really understand why it should be FlowDataID here ?  I think everytime the stage runs ,it depends on data